packages/components/package.json

```diff
@@ -133,5 +133,9 @@
 		"doc",
 		"custom-elements.json",
 		"vscode-custom-data.json"
-	]
+	],
+	"lint-staged": {
+		"*.ts|tsx|js|jsx": ["prettier --list-different src", "pnpm lint"],
```

Comment:
Ich glaube, dass wir auch noch css, scss und md möglich sind. Könnten wir das noch prüfen?

Reply:
Im allgemeinen konfigurieren wir Tools in separaten Dateien, nicht in der package.json. Ich würde das auch hier gerne so fortführen, und die Konfiguration in eine .lintstagedrc verschieben: https://github.com/lint-staged/lint-staged?tab=readme-ov-file#configuration